### PR TITLE
Polish MD: Misc files

### PR DESCRIPTION
**Pull request by Jay-o-Way**
Linked: #8352
Part 1 of 8.

### CODE REVIEW

File: samples/RaspberryPiInstructions.md [hunk 1]
```diff
@@ -86,8 +84,8 @@ Pete Gallagher has also created single line install scripts for .NET Core and .N
 
 Please refer to Pete Gallagher's Blog;
 
-- [.NET Core](https://bit.ly/dotnetcorepi)
-- [NET 5](https://bit.ly/dotnet5pi)
+* [.NET Core](https://bit.ly/dotnetcorepi)
```
Comment (mairaw) [inline samples/RaspberryPiInstructions.md:87]: in one file, you replace - with *, and here you did the opposite. what's the preferred one?

Comment (Jay-o-Way) [reply samples/RaspberryPiInstructions.md:87]: This is a result of the VS Code, automatically applying the first used char on all lists.

Comment (mairaw) [reply samples/RaspberryPiInstructions.md:87]: Gotcha.

**Review by mairaw — APPROVED** [review]
Thanks!

**Pull request merged**